core/src/main/java/com/linecorp/armeria/client/ClientOptions.java

```diff
@@ -71,8 +70,8 @@
 
     private static final ClientOptionValue<?>[] DEFAULT_OPTIONS = {
             DEFAULT_WRITE_TIMEOUT_MILLIS.newValue(DEFAULT_DEFAULT_WRITE_TIMEOUT_MILLIS),
-            DEFAULT_RESPONSE_TIMEOUT_MILLIS.newValue(DEFAULT_DEFAULT_RESPONSE_TIMEOUT_MILLIS),
-            DEFAULT_MAX_RESPONSE_LENGTH.newValue(DEFAULT_DEFAULT_MAX_RESPONSE_LENGTH),
+            DEFAULT_RESPONSE_TIMEOUT_MILLIS.newValue(Flags.defaultRequestResponseTimeoutMillis()),
+            DEFAULT_MAX_RESPONSE_LENGTH.newValue(Flags.defaultMaxRequestResponseLength()),
```

Comment:
I agree with you on that having defaultResponseTimeoutMillis greater than defaultRequestTimeoutMillis. I just wanted not to break the the old values.
In respect of maximum request and response length, I thought that they have the same value and it would be better  if the values are from same variable in `Flags`... But I'm not sure. Is there any reason that they need to be separated?